pkgs/development/ruby-modules/gem-config/default.nix

```diff
@@ -394,10 +394,6 @@ in
     '';
   };
 
-  mimemagic = attrs: {
```

Comment:
It looks like there were 2 of these locally so I removed the oldest

Reply:
This was fixed in master yesterday. You should have rebased onto origin/master instead of fixing this yourself.

Reply:
Rebased to fix this